src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -368,6 +368,14 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
             }
 
             CheckModifiers(location, diagnostics);
+
+            foreach (var typeParameter in _typeParameters)
+            {
+                if (typeParameter.HasUnmanagedTypeConstraint)
```

Comment:
>if (typeParameter.HasUnmanagedTypeConstraint) [](start = 16, length = 45)

This check belong in completion for the type parameter, this feels like a wrong place for this code.

Reply:
Since we decided to enable local functions, the last parameter would be false. Will add that now.

---
In reply to: [172418067](https://github.com/dotnet/roslyn/pull/24817#discussion_r172418067) [](ancestors = 172418067)

Reply:
I thin the added code belongs in completion for the type parameter

---
In reply to: [175998583](https://github.com/dotnet/roslyn/pull/24817#discussion_r175998583) [](ancestors = 175998583,172418067)

Reply:
Consolidated into the other comment.

---
In reply to: [177244069](https://github.com/dotnet/roslyn/pull/24817#discussion_r177244069) [](ancestors = 177244069,175998583,172418067)